crates/db_connection_pool/src/postgrespool.rs

```diff
@@ -38,31 +38,32 @@ impl PostgresConnectionPool {
         params: Arc<Option<HashMap<String, String>>>,
         secret: Option<Secret>,
     ) -> Result<Self> {
-        let mut host = "localhost";
-        let mut user = "postgres";
-        let mut dbname = "postgres";
-        let mut connection_string = String::new();
+        let mut connection_string = "host=localhost user=postgres dbname=postgres".to_string();
 
         if let Some(params) = params.as_ref() {
-            if let Some(pg_host) = params.get("pg_host") {
-                host = pg_host;
-            }
-            if let Some(pg_user) = params.get("pg_user") {
-                user = pg_user;
-            }
-            if let Some(pg_db) = params.get("pg_db") {
-                dbname = pg_db;
-            }
-            if let Some(pg_pass) = get_pg_pass(params, secret) {
-                connection_string.push_str(format!("password={pg_pass} ").as_str());
-            }
-            if let Some(pg_port) = params.get("pg_port") {
-                connection_string.push_str(format!("port={pg_port} ").as_str());
+            connection_string = String::new();
+
+            if let Some(pg_connection_string) = params.get("pg_connection_string") {
```

Comment:
This should also be configurable via secrets

Reply:
Good call